core/storage/hamt/hamt.cpp

```diff
@@ -88,6 +88,10 @@ namespace fc::storage::hamt {
 
   outcome::result<CID> Hamt::flush() {
     OUTCOME_TRY(flush(root_));
+    return cid();
```

Comment:
Why? You will always use cid after flush.

Reply:
Left for compatibility.
New code will not create new names for flushed cid and just refer amt/hamt object.